components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Engine/LegacyJobOrchestrator.cs

```diff
@@ -116,11 +117,13 @@ IJsonSerializer jsonSerializer
 
         /// <summary>
         /// Receives the heartbeat from the LegacyJobOrchestrator, JobProcess; used to control lifetime of job (cancel, restart, keep).
+        /// Used also to receive the diagnostic info
         /// </summary>
         /// <param name="heartbeat"></param>
+        /// <param name="info"></param>
         /// <param name="ct"></param>
         /// <returns></returns>
-        public async Task<HeartbeatResultModel> SendHeartbeatAsync(HeartbeatModel heartbeat, CancellationToken ct = default) {
+        public async Task<HeartbeatResultModel> SendHeartbeatAsync(HeartbeatModel heartbeat, JobDiagnosticInfoModel info, CancellationToken ct = default) {
```

Comment:
This line is too long, please break it down. And in general, we should not have lines longer than 120 characters.

Reply:
changed